dockerfiles/Dockerfile3.cpu

```diff
@@ -0,0 +1,44 @@
+# Dockerfile for CPU-only CakeChat setup
+
+FROM ubuntu:16.04
+
+# TensorBoard-related packages versions
+ARG TENSORBOARD_VERSION=1.5.0
+ARG TENSORFLOW_VERSION=1.5.0
+
+# Install some dependencies
+RUN apt-get update && apt-get install -y \
+        curl \
+        git \
+        screen \
+        tmux \
+        sudo \
+        nano \
+        pkg-config \
+        software-properties-common \
+        unzip \
+        vim \
+        wget \
+        python3 \
+        python3-dev \
+        python3-pip \
+        python3-setuptools \
+        && \
+    apt-get clean && \
+    apt-get autoremove && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install up-to-date pip
+RUN pip3 --no-cache-dir install -U pip
```

Comment:
After I ran this command in the existing container with python2, I've got the following error

```
Traceback (most recent call last):
  File "/usr/bin/pip3", line 9, in <module>
    from pip import main
ImportError: cannot import name 'main'
```

Here's more info about this: https://stackoverflow.com/questions/49836676/python-pip3-cannot-import-name-main-error-after-upgrading-pip

Probably there is a more stable way to upgrade pip here that we should use. 
However, the container builds and runs correctly and everything seems to be fine.

Reply:
Conclusion: if there is a more stable way to upgrade here, probably we should use it.
Otherwise, we can keep it like that.